### ISSUE TITLE
Move CI from Travis to Centos CI

### ISSUE THREAD

**Issue by jpopelka** [user-cont/release-bot#105]
See #104

**Comment by TomasTomecek** [comment]
Is this done?

**Comment by jpopelka** [comment]
@rpitonak where are we? I see we have:
* [Jenkinsfile](https://github.com/user-cont/release-bot/blob/master/Jenkinsfile)
* [release-bot-push](https://ci.centos.org/view/UserCont/job/release-bot-push) and [release-bot-pr](https://ci.centos.org/view/UserCont/job/release-bot-pr) jobs

I guess we still need to figure out the secrets and also I see that the release-bot-pr was run the last time on Mar 11th.

**Comment by rpitonak** [comment]
> I guess we still need to figure out the secrets and also I see that the release-bot-pr was run the last time on Mar 11th.

Exactly, we need to solve the secrets or change the test suite to not use github directly.
